sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from sympy.core.singleton import S
```

Comment:
Using relative import gives the following error for python3 in travis build [here](https://travis-ci.org/sympy/sympy/jobs/143955456):
```python
Exception occurred:
  File "../sympy/sets/__init__.py", line 7, in <module>
    from ..core.singleton import S
ValueError: attempted relative import beyond top-level package
```